### PR TITLE
CI improvements

### PR DESCRIPTION
**Pull request by Waldz**

### CODE REVIEW

File: bin/builder_run [hunk 1]
```diff
@@ -23,5 +23,5 @@ if [ -z "$COMMAND" ]; then
 fi
 
 docker run --rm -ti \
-    -v $(pwd):/go/src/github.com/mysteriumnetwork/node mysteriumnetwork/mysterium-node-builder \
-    -c "cd /go/src/github.com/mysteriumnetwork/node && ${COMMAND}"
+    -v $(pwd):/go/src/github.com/mysteriumnetwork/node \
```
Comment (soffokl) [inline bin/builder_run]: Default working directory is `/go/src/github.com/mysterium/node` which does not exist at the moment.
We can rebuild docker image or just change the working directory during the start time.

Comment (Waldz) [reply bin/builder_run]: Yes, I rebuilt `builder` image

Comment (soffokl) [reply bin/builder_run]: hm, but it failed on CI

Comment (Waldz) [reply bin/builder_run]: Failed to push. Fixed

**Pull request merged**